### PR TITLE
refac: Simplify settlement_report.metering_point_time_series_v1 view

### PR DESCRIPTION
**Pull request by JesperGomes**
# Description

To improve performance it is is better to not do the group by inside the view, but rather do it on the caller side.
Therefore the daylight savings feature tests are no longer needed.

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_settlement_report_metering_point_time_series_v1.sql
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_settlement_report_metering_point_time_series_v1.sql:1]: How about adding code doc about the learnings - so future developers don't need to consider doing what we did up until now?

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_settlement_report_metering_point_time_series_v1.sql [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
+       t.observation_time,
+       t.quantity
 FROM {BASIS_DATA_DATABASE_NAME}.metering_point_periods AS m
   INNER JOIN {BASIS_DATA_DATABASE_NAME}.calculations AS c ON c.calculation_id = m.calculation_id
   INNER JOIN {BASIS_DATA_DATABASE_NAME}.time_series_points AS t ON m.metering_point_id = t.metering_point_id AND m.calculation_id = t.calculation_id
 WHERE c.calculation_type IN ('balance_fixing', 'wholesale_fixing', 'first_correction_settlement', 'second_correction_settlement', 'third_correction_settlement')
   AND t.observation_time >= m.from_date
   AND (m.to_date IS NULL OR t.observation_time < m.to_date)
-GROUP BY
-    c.calculation_id, m.metering_point_id, m.metering_point_type, DATE_TRUNC('day', FROM_UTC_TIMESTAMP(t.observation_time, 'Europe/Copenhagen')), m.resolution, m.grid_area_code, m.energy_supplier_id
```
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_settlement_report_metering_point_time_series_v1.sql:19]: Is this a wild guess or do you have some data on the performance difference?

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202406271000_Modify_settlement_report_metering_point_time_series_v1.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
+DROP VIEW IF EXISTS {SETTLEMENT_REPORT_DATABASE_NAME}.metering_point_time_series_v1
+GO
+
+CREATE VIEW IF NOT EXISTS {SETTLEMENT_REPORT_DATABASE_NAME}.metering_point_time_series_v1 AS
```
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/package/datamigration/migration_scripts/202406271000_Modify_settlement_report_metering_point_time_series_v1.sql:4]: I assume that you have agreed with Raccoons to deliver this breaking change in place.

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_settlement_report_metering_point_time_series_v1.sql (hunk 1 above)
Comment (JesperGomes) [reply source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_settlement_report_metering_point_time_series_v1.sql:19]: Raccoons have made some experiements which resulted in this conclusion

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202406271000_Modify_settlement_report_metering_point_time_series_v1.sql (hunk 2 above)
Comment (JesperGomes) [reply source/databricks/calculation_engine/package/datamigration/migration_scripts/202406271000_Modify_settlement_report_metering_point_time_series_v1.sql:4]: yes

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**